test/fixtures/linter/autofix/ansi/016_index_error_with_jinja_if/after.sql

```diff
@@ -0,0 +1,16 @@
+-- This file combines product data from individual brands into a staging table
+{% set products =  [
+  'table1',
+  'table2'] %}
+
+{% for product in products %}
+SELECT
+    brand,
+    country_code,
+    category,
+    name,
+    id
+FROM
+    product
+{% if not loop.last -%} UNION ALL {%- endif %}
```

Comment:
Can we also add a test case as well without the `-%}`? I.e. with:

```jinja
{% if not loop.last %} UNION ALL {% endif %}
```

And also another test case with something afterwards (a `;` or even an `ORDER BY`)? When you add this, this causes the middle path (`ts_start_sf_start < len(self.sliced_file)`) to be covered (hence no need for `pragma: no cover` in there!).

Reply:
Added this in https://github.com/sqlfluff/sqlfluff/pull/1430/commits/62687a79887205a0c877ccdd8bf18be44e39738e